airbyte-integrations/connectors/source-freshsales/source_freshsales/schemas/contacts.json

```diff
@@ -60,6 +61,24 @@
     "work_email": { "type": ["null", "string"] },
     "customer_fit": { "type": ["null", "integer"] },
     "whatsapp_subscription_status": { "type": ["null", "integer"] },
-    "phone_numbers": { "type": ["null", "array"] }
+    "phone_numbers": { "type": ["null", "array"] },
+    "first_seen_chat": { "type": ["null", "string"] },
+    "latest_campaign": { "type": ["null", "string"] },
+    "latest_source": { "type": ["null", "string"] },
+    "unsubscription_reason": { "type": ["null", "string"] },
+    "first_campaign": { "type": ["null", "string"] },
+    "total_sessions": { "type": ["null", "string"] },
+    "mcr_id": { "type": ["null", "string"] },
+    "last_campaign": { "type": ["null", "string"] },
+    "last_medium": { "type": ["null", "string"] },
+    "last_seen_chat": { "type": ["null", "string"] },
+    "first_medium": { "type": ["null", "string"] },
+    "other_unsubscription_reason": { "type": ["null", "string"] },
+    "system_tags": { "type": ["null", "string"] },
+    "latest_medium": { "type": ["null", "string"] },
+    "first_source": { "type": ["null", "string"] },
+    "sms_subscription_status": { "type": ["null", "string"] },
+    "locale": { "type": ["null", "string"] },
+    "last_source": { "type": ["null", "string"] }
```

Comment:
I didn't double check the types against the API when reviewing, most seem reasonable though